<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Построчный алгоритм заполнения с затравкой</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет заполнения</string>
    </property>
    <widget class="QComboBox" name="box_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Зеленый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Черный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Розовый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Бирюзовый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Голубой</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Красный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Синий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Желтый</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим</string>
    </property>
    <widget class="QRadioButton" name="but_delay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>С задержкой</string>
     </property>
    </widget>
    <widget class="QSlider" name="delay_lvl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
     <property name="sliderPosition">
      <number>15</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Добавление точки</string>
    </property>
    <widget class="QDoubleSpinBox" name="box_y">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="prefix">
      <string>Y: </string>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="box_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="prefix">
      <string>X: </string>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="but_add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Вершина</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_seed">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Затравочная</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graph">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>950</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Введенные точки</string>
    </property>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="but_fill">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрасить фигуру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clean">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_time">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>820</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Время:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_connect">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Замнкнуть контур</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет границы</string>
    </property>
    <widget class="QComboBox" name="box_color_border">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Черный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Зеленый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Розовый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Бирюзовый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Голубой</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Красный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Синий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Желтый</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>graph</zorder>
   <zorder>but_fill</zorder>
   <zorder>but_clean</zorder>
   <zorder>lbl_time</zorder>
   <zorder>but_connect</zorder>
   <zorder>groupBox_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
